src/modules/posts/components/PostProfileView.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import { Container, Grid } from '@material-ui/core';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import BorderBottom from '../../../lib/components/Border';
+import LoadingInfo from '../../../lib/components/LoadingInfo';
+import { RootStateType } from '../../../store/rootReducer';
+import { fetchPostById } from '../store/postsSlice';
+import PostInfo from './PostInfo';
+import { useStyles } from '../styles/PostProfileView.styles';
+
+const PostProfileView: React.FC = () => {
+  const classes = useStyles();
+  const { postId } = useParams<{ postId: string }>();
+  const {
+    posts,
+    meta: { loading },
+  } = useSelector((state: RootStateType) => state.posts.posts);
+  const dispatch = useDispatch();
```

Comment:
that's not good..
a single post component should not know anything about all loaded posts.

so let's siplify the stuff here:
- just parse the `id` from the url and get the post and display it
- if there's no post with such an id show 404 error (for now, might be a more informative message latter)
- don't use store to store all loaded posts 
- don't store it in the redux for now

let's not use caching in any form for now, for just displaying the post it looks like obsolete